web-service-consumer/1.5/modules/ROOT/pages/index.adoc

```diff
@@ -1,5 +1,4 @@
 = Web Service Consumer Connector- Mule 4
-:page-aliases: connectors::web-service/web-service-consumer.adoc
 
```

Comment:
Important: We are no longer using RELEASE in the .pom file example because one of the Gastons told us it was a security issue. If you look at the latest template for the XML topic, you will see different wording for the version.

Minor comments that can wait:
For the list below, remove the periods. (I couldn't add a comment there.)
I think "Headers," "Body," and "Attachment" should be lowercase as in  "SOAP Headers," etc.
'Unified" should be lowercase.
"as a transport" instead of "as transport"
We are adding links to two intro topics in the See Also area. The template has these links. Also we are splitting out the XML Maven info into it's own topic.